### PR TITLE
Spaces

### PR DESCRIPTION
**Pull request by RobertsK284**
Hi, we have finished back end work in the spaces model.

Would you please check and comment if you agree this can be merged?

### CODE REVIEW

File: spec/models/space_spec.rb [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+describe Space do
+  let(:user_db_params) do
+    { name: 'Name1', username: 'Username',
+      email: 'email@email.com', password: 'qwe' }
+  end
+  let(:space_obj_params) do
+    { id: '1', name: 'Flat1',
+      description: '1-bedroom flat',
+      date_available: '2019-01-01', booked: 'f',
```
Comment (willmcb) [inline spec/models/space_spec.rb:12]: I think this should be `'false'` rather than `'f'` throughout

**Review by willmcb — COMMENTED** [review]
See comments

**Comment by willmcb** [comment]
:shipit:

**Comment by Ajay233** [comment]
:shipit:

**Pull request merged**